### PR TITLE
[CI] qwen image edit L4 accuracy test

### PR DESCRIPTION
**Pull request by fhfuih**
PLEASE FILL IN THE PR DESCRIPTION HERE ENSURING ALL CHECKLIST ITEMS (AT THE BOTTOM) HAVE BEEN CONSIDERED.

## Purpose

Self-explained.

Per offline discussion with @david6666666 , this PR supersedes #2714

## Test Plan

Using SSIM and PSNR to compare the results

## Test Result

Running on my local side first... Will request a ready tag and attach results once done

---
<details>
<summary> Essential Elements of an Effective PR Description Checklist </summary>

- [x] The purpose of the PR, such as "Fix some issue (link existing issues this PR will resolve)".
- [x] The test plan. Please provide the test scripts & test commands. Please state the reasons if your codes don't require additional test scripts. For test file guidelines, please check the [test style doc](https://docs.vllm.ai/projects/vllm-omni/en/latest/contributing/ci/tests_style/)
- [ ] The test results. Please paste the results comparison before and after, or the e2e results.
- [ ] (Optional) The necessary documentation update, such as updating `supported_models.md` and `examples` for a new model. **Please run `mkdocs serve` to sync the documentation editions to `./docs`.**
- [ ] (Optional) Release notes update. If your change is user-facing, please update the release notes draft.
</details>

**BEFORE SUBMITTING, PLEASE READ <https://github.com/vllm-project/vllm-omni/blob/main/CONTRIBUTING.md>** (anything written below this line will be removed by GitHub Actions)

### CODE REVIEW

File: tests/e2e/accuracy/test_qwen_image_edit.py [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+            image = Image.open(BytesIO(response.content)).convert("RGB")
+            image.load()
+            inputs.append(image)
+        input_image = inputs[0] if len(inputs) == 1 else inputs
+        pipe = DiffusionPipeline.from_pretrained(
+            model,
+            torch_dtype=torch.bfloat16,
+            trust_remote_code=True,
+        ).to("cuda")
+        generator = torch.Generator(device="cuda").manual_seed(SEED)
```
[the remark this answers is not in the document]
Comment (fhfuih) [reply tests/e2e/accuracy/test_qwen_image_edit.py]: Resolved

File: tests/e2e/accuracy/utils.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    height: int,
+    ssim_threshold: float,
+    psnr_threshold: float,
+) -> None:
+    requested_size = (width, height)
+    if diffusers_image.size != requested_size:
+        pytest.skip(
+            "Diffusers output dimensions do not match requested size; "
+            f"requested={requested_size}, diffusers={diffusers_image.size}"
+        )
```
[the remark this answers is not in the document]
Comment (fhfuih) [reply tests/e2e/accuracy/utils.py:34]: This is by design: When the baseline (diffusers) is corrupted, it is meaningless to compare. Leaving this open---please comment if anyone thinks this is NOT OK.

File: tests/e2e/accuracy/test_qwen_image_edit.py [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+    prompt: str,
+    input_image_urls: list[str],
+    output_path: Path,
+) -> Image.Image:
+    _run_pre_test_cleanup(enable_force=True)
+    pipe: Any = None
+    try:
+        inputs: list[Image.Image] = []
+        for url in input_image_urls:
+            response = requests.get(url, timeout=60)
```
[the remark this answers is not in the document]
Comment (fhfuih) [reply tests/e2e/accuracy/test_qwen_image_edit.py]: Fixed

**Comment by david6666666** [comment]
add test to test-nightly.yml

**Comment by david6666666** [comment]
```
<html>
<body>
<!--StartFragment-->
Qwen/Qwen-Image-Edit similarity metrics:
--
SSIM: value=0.991259, threshold>=0.940000, range=[-1, 1], higher_is_better=True
PSNR: value=40.783478 dB, threshold>=28.000000 dB, range=[0, +inf), higher_is_better=True
PASSEDGPU cleanup disabled
 
tests/e2e/accuracy/test_qwen_image_edit.py::test_qwen_image_edit_multiple_matches_diffusers SKIPPED (Skipping as the second image seems to be ignore...)
 
======================================================================= warnings summary =======================================================================
<frozen importlib._bootstrap>:488
<frozen importlib._bootstrap>:488: DeprecationWarning: builtin type SwigPyPacked has no __module__ attribute
 
<frozen importlib._bootstrap>:488
<frozen importlib._bootstrap>:488: DeprecationWarning: builtin type SwigPyObject has no __module__ attribute
 
../../../usr/local/lib/python3.12/dist-packages/torch/jit/_script.py:362: 14 warnings
/usr/local/lib/python3.12/dist-packages/torch/jit/_script.py:362: DeprecationWarning: `torch.jit.script_method` is deprecated. Please switch to `torch.compile` or `torch.export`.
warnings.warn(
 
vllm_omni/__init__.py:28
/workspace/build/buildkite/vllm_omni/__init__.py:28: RuntimeWarning: Failed to import version from _version.py: No module named 'vllm_omni._version'
This typically happens in development mode before building.
Using fallback version 'dev'.
from .version import __version__, __version_tuple__  # isort:skip
 
-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
==================================================== 1 passed, 1 skipped, 17 warnings in 264.30s (0:04:24) =====================================================

<!--EndFragment-->
</body>
</html>
```

**Comment by david6666666** [comment]
LGTM

**Review by david6666666 — APPROVED** [review]
